src/components/Layout/HeaderDropDown.js

```diff
@@ -30,14 +30,19 @@ class HeaderDropDown extends React.Component {
 
   render () {
     return (
-      <div className={cx(this.props.className, style.dropDownItem)}>
+      <a className={cx(this.props.className, style.dropDownItem)}>
```

Comment:
not quite happy with that either, but NavElement renders as Anchor and browsers have default values for many css-props - so the lang dropdown was slightly bigger than the other nav-elements because of these default-values. To avoid long messy css - props I set this as an anchor